extension-config/popup.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>tota11y settings</title>
+  <link href='//fonts.googleapis.com/css?family=Roboto:300,400,500,700' rel='stylesheet' type='text/css' />
+  <style>
+    body {
+      font-family: 'Roboto', sans-serif;
+      width: 16rem;
+    }
+
+    .settings {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: space-between;
+    }
+
+    .settings>* {
+      margin-bottom: 1rem;
+    }
+  </style>
+</head>
+
+<body>
+  <div class="settings">
+    <img src="assets/icon-128.png" />
+    <a href="https://khan.github.io/tota11y" target="_blank" rel="noopener">tota11y</a>
+    <p>An accessibility visualization toolkit</p>
+    <label for="tota11y-toggler">Show tota11y<input id="tota11y-toggler" type="checkbox" /></label>
```

Comment:
If this is already showing, should it say "Hide tota11y"? Or, just change it to `Toggle tota11y`?

Reply:
Initially tota11y is not showing. I'll update the text to `Toggle tota11y`.